app/components/press-release/files-table.hbs

```diff
@@ -0,0 +1,55 @@
+<div class="auk-scroll-wrapper">
+  <div class="auk-scroll-wrapper__body">
+    <div class="auk-u-my-4">
+      {{#if @model.length}}
+        <table class="auk-table">
+          <thead>
+            <tr>
+              <th>
+                Naam bijlage
+              </th>
+              <th>
+                Grootte bestand
+              </th>
+              <th>
+                Type bestand
+              </th>
+              <th>
+                {{!-- remove source --}}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {{#each (sort-by "name" @model) as |row|}}
+              <tr>
+                <td class="auk-u-text-nowrap">
+                  {{row.name}}
+                </td>
+                <td>
+                  {{row.humanReadableSize}}
+                </td>
+                <td>
+                  {{row.format}}
```

Comment:
Hier kan je beter `row.extension` gebruiken.
In Kaleidos is er een component die je kan overnemen om de extensie weer te geven: https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/development/app/components/utils/documents/file-type-pill.js